ivadomed/main.py

```diff
@@ -189,8 +202,31 @@ def run_command(context, n_gif=0, thr_increment=None):
             thr_increment=thr_increment,
             debugging=context["debugging"])
 
-        # Update threshold in config file
         if thr_increment:
+            # Choice of optimisation metric
```

Comment:
What do you think about putting all this block directly in testing, this way we don't need to retrain to have an optimal threshold?